xml/bp_troubleshooting_logging.xml

```diff
@@ -1472,4 +1472,25 @@ debug mds balancer = 1/20
     </variablelist>
   </sect2>
  </sect1>
+
+ <sect1 xml:id="logging-kernelrbd-cephfs-clients">
+   <title>Kernel RBD and &cephfs; clients</title>
+   <para>
+     The RBD and &cephfs; kernel clients provide means to enable runtime
+     logging that allow low-level debugging.  However, the performance penalty
+     may be high and that is why dynamic debugging is turned off by default.
+     Also, analysing these logs requires comparing them with the kernel source
+     code.
+   </para>
+   <para>
+     For example, to see aecho "file fs/ceph/snap.c +pf" > /sys/kernel/debug/dynamic_debug/controlll the debug messages related with &cephfs;
+     snapshot handing, you could try the following:
+   </para>
+<screen>&prompt.root;echo "file fs/ceph/snap.c +pf" > /sys/kernel/debug/dynamic_debug/control</screen>
+ </sect1>
+   <para>
+     The kernel logs would now include any debug message in file <filename>fs/ceph/snap.c</filename>.
+     In order to disable the dynamic debug:
+   </para>
+<screen>&prompt.root;echo "file fs/ceph/snap.c +pf" > /sys/kernel/debug/dynamic_debug/control</screen>
```

Comment:
My mistake here: I gave you the wrong command.  The correct command to disable dynamic debug would be:

echo "file fs/ceph/snap.c -p" > /sys/kernel/debug/dynamic_debug/control